enterprise/frontend/src/metabase-enterprise/rich_text_editing/tiptap/extensions/CardEmbed/CardEmbedNode.tsx

```diff
@@ -150,26 +196,37 @@ export const CardEmbedComponent = memo(
     const { data: commentsData } = useListCommentsQuery(
       getListCommentsQuery(document),
     );
+
     const comments = commentsData?.comments;
     const hasUnsavedChanges = useSelector(getHasUnsavedChanges);
-    const [hovered, setHovered] = useState(false);
     const { _id } = node.attrs;
     const isOpen = childTargetId === _id;
     const isHovered = hoveredChildTargetId === _id;
     const threads = useMemo(
       () => getTargetChildCommentThreads(comments, _id),
       [comments, _id],
     );
-    const { refs, floatingStyles } = useFloating({
-      placement: "right-start",
-      whileElementsMounted: autoUpdate,
-      strategy: "fixed",
+    const unresolvedCommentsCount = useMemo(
+      () => getUnresolvedComments(threads).length,
+      [threads],
+    );
+    const {
+      component, // don't use Link component here since it messes with tiptap's link handling
```

Comment:
Link was used for proper a11y.
TipTap's interference is the reason why `CommentsMenu` uses `createPortal`.
We could do the same thing here.